packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/style/Gradient.kt

```diff
@@ -21,63 +20,46 @@ public class Gradient(gradient: ReadableMap?, context: Context) {
   }
 
```

Comment:
Can we make this class `internal` instead of public. We don' want people to start relying on this yet